plugin/assets/css/src/components/card.css

```diff
@@ -192,6 +192,7 @@
 	color: var(--md-sys-color-on-surface, #000);
 }
 
+.entry-content h2.post-card__title,
```

Comment:
Entry content adds h2 padding by default, This is resetting that for material card for query loop.